diff -ruN esmf-ESMF_8_1_1.orig/src/addon/ESMPy/setup.py esmf-ESMF_8_1_1/src/addon/ESMPy/setup.py
--- esmf-ESMF_8_1_1.orig/src/addon/ESMPy/setup.py	2021-04-23 00:21:45.000000000 +0200
+++ esmf-ESMF_8_1_1/src/addon/ESMPy/setup.py	2021-09-21 17:52:19.798760790 +0200
@@ -242,7 +242,7 @@
 # TODO: build doc command
 # TODO: remove duplicated metadata: here and src/ESMF/__init__.py
 setup(name="ESMPy",
-      version="8.1.0",
+      version="8.1.1",
       description="ESMF Python interface",
       author="University Corporation for Atmospheric Research, \
               Massachusetts Institute of Technology, \
