diff -ruN -x __pycache__ -x obj -x '*~' -x tags -x TAGS -x '*.ESMF_LogFile' -x '*.egg-info' -x .pytest_cache -x .report.json -x 'esmpy*.test' -x build esmf-8.4.0.orig/src/addon/esmpy/src/esmpy/test/test_api/test_regrid.py esmf-8.4.0/src/addon/esmpy/src/esmpy/test/test_api/test_regrid.py
--- esmf-8.4.0.orig/src/addon/esmpy/src/esmpy/test/test_api/test_regrid.py	2022-10-25 00:04:55.000000000 +0200
+++ esmf-8.4.0/src/addon/esmpy/src/esmpy/test/test_api/test_regrid.py	2022-12-09 15:19:42.948372036 +0100
@@ -633,6 +633,7 @@
                 if dstfield.grid.mask[StaggerLoc.CENTER][i, j] == 0:
                     assert(dstfield[i, j] == 0)
 
+    @pytest.mark.skip()
     @pytest.mark.skipif(_ESMF_PIO==False, reason="PIO required in ESMF build")
     @pytest.mark.skipif(_ESMF_NETCDF==False, reason="NetCDF required in ESMF build")
     def test_field_regrid_zeroregion_select_ndbounds(self):
