diff -ruN -x __pycache__ -x obj -x '*~' -x tags -x TAGS -x autom4te.cache esmf-8.4.2.orig/src/addon/esmpy/pyproject.toml esmf-8.4.2/src/addon/esmpy/pyproject.toml
--- esmf-8.4.2.orig/src/addon/esmpy/pyproject.toml	2023-05-22 14:11:38.426907629 +0200
+++ esmf-8.4.2/src/addon/esmpy/pyproject.toml	2023-05-22 14:12:47.777200130 +0200
@@ -11,7 +11,6 @@
 license = {text = "University of Illinois-NCSA"}
 dependencies = [
     "numpy",
-    "pytest-json-report",
     'importlib-metadata; python_version < "3.8"',
     # setuptools-git-versioning shouldn't be needed here, but is
     # included as a workaround for problems with the build-time
@@ -23,6 +22,12 @@
 ]
 dynamic = ["version"]
 
+[project.optional-dependencies]
+testing = [
+    "pytest",
+    "pytest-json-report",
+]
+
 [tool.setuptools-git-versioning]
 enabled = true
 template = "{tag}"
