diff -ruN -x __pycache__ -x obj -x '*~' -x tags -x TAGS -x '*.ESMF_LogFile' -x '*.egg-info' -x .pytest_cache -x .report.json -x 'esmpy*.test' -x build esmf-8.4.0.test/src/addon/esmpy/pyproject.toml esmf-8.4.0/src/addon/esmpy/pyproject.toml
--- esmf-8.4.0.test/src/addon/esmpy/pyproject.toml	2023-02-06 14:07:07.369242226 +0100
+++ esmf-8.4.0/src/addon/esmpy/pyproject.toml	2023-02-06 14:07:12.881277255 +0100
@@ -6,7 +6,7 @@
 name = "esmpy"
 description = "ESMF Python interface"
 # readme = "README.md"
-maintainers = [{author = "ESMF Core Team", email = "esmf_support@ucar.edu"}]
+maintainers = [{name = "ESMF Core Team", email = "esmf_support@ucar.edu"}]
 requires-python = ">=3.7"
 license = {text = "University of Illinois-NCSA"}
 dependencies = [
